### ISSUE TITLE
Telegram bridge not working

### ISSUE THREAD

**Issue by timo95** [spantaleev/matrix-docker-ansible-deploy#202]
Log:
```
matrix-synapse[17797]: /usr/bin/docker: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /matrix/mautrix-telegram/registration.yaml.
```

matrix-synapse.service:
```
--mount type=bind,src=/matrix/mautrix-telegram/registration.yaml,dst=/matrix-mautrix-telegram-registration.yaml,ro \
```

**Comment by spantaleev** [comment]
Likely a regression since 4e8543c.

I suppose it's fixed in 169b09f0edc21a42

**Comment by timo95** [comment]
It works again
